src/vivarium_csu_hypertension_sdc/components/utilities.py

```diff
@@ -185,3 +185,39 @@ def get_therapeutic_inertia_probability(mean, sd, randomness):
     np.random.seed(seed)
     draw = np.random.random()
     return dist.ppf(draw)
+
+
+def choose_half_dose_single_new_drug(index, med_probabilities, randomness, current_dosages):
```

Comment:
Is there a reason these are not methods on your Treatment algorithm?  They seem like they should belong to the class, even just as static methods.

Reply:
nope. I just got overzealous in moving everything off into utilities. I'll put them back